### PR TITLE
More info about destination interface in Node

### PR DESCRIPTION
**Pull request by alex-tpom6oh**

### CODE REVIEW

**Comment by hugihlynsson** [comment]
#### 📝 Changed routes:


| [**…n/reference/javascript**](https://docs-fbdgtb6wu.avo-dev.app/implementation/reference/javascript) | [**…n/reference/typescript**](https://docs-fbdgtb6wu.avo-dev.app/implementation/reference/typescript) |
|:-:|:-:|
|<a href="https://docs-fbdgtb6wu.avo-dev.app/implementation/reference/javascript">Screenshot of /implementation/reference/javascript</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/iqKy5saQEESt22ZIbXP8EhkRm143OQV_d0E-51EAd41cPuJ5JIdNx-o_RTFHshkpBFsQxmRJbr1tRIahIAtdTw.png">(view full size)</a> |<a href="https://docs-fbdgtb6wu.avo-dev.app/implementation/reference/typescript">Screenshot of /implementation/reference/typescript</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/2i5Et2sc15iNpK1SuceEmUrw8U7OPXuAKl85RUBysvWD9l9qOEJXZf0HeO605JYxanfx8Z2GjjrRSWL0FHbUWA.png">(view full size)</a> |







Commit a875a557cb1bbcfce40b8672cd6b40f9c3fa7add (https://docs-fbdgtb6wu.avo-dev.app).

File: pages/implementation/reference/typescript.mdx [hunk 1]
```diff
@@ -223,6 +223,8 @@ let customDestination = {
 };
 ```
 
+Find destination interface snippets for different platforms and analytics destinations [in our GitHub])("https://github.com/avohq/destination-interface-code-snippets/tree/main/typescript").
```
Comment (alex-tpom6oh) [inline pages/implementation/reference/typescript.mdx:226]:
```suggestion
Find destination interface snippets for different platforms and analytics destinations [in our GitHub]("https://github.com/avohq/destination-interface-code-snippets/tree/main/typescript").
```

File: pages/implementation/reference/typescript.mdx [hunk 2]
```diff
@@ -223,6 +223,8 @@ let customDestination = {
 };
 ```
 
+Find destination interface snippets for different platforms and analytics destinations [in our GitHub]("https://github.com/avohq/destination-interface-code-snippets/tree/main/typescript").
```
Comment (alex-tpom6oh) [inline pages/implementation/reference/typescript.mdx:226]:
```suggestion
Find destination interface snippets for different platforms and analytics destinations [in our GitHub](https://github.com/avohq/destination-interface-code-snippets/tree/main/typescript).
```

File: pages/implementation/reference/typescript.mdx [hunk 3]
```diff
@@ -150,7 +150,7 @@ Custom destination interface format:
 };
 ```
 
-In Node each callback gets an additional first parameter - `userId` and, optionally, a `anonymousId` parameter. To get the `anonymousId` parameter in your workspace [reach out to us](https://www.avo.app/docs/help/troubleshooting).
+In Node each callback is asynchronous and gets an additional first parameter - `userId` and, optionally, a `anonymousId` parameter. To get the `anonymousId` parameter in your workspace [reach out to us](https://www.avo.app/docs/help/troubleshooting).
```
Comment (logason) [inline pages/implementation/reference/typescript.mdx:153]: Should we add something more about this here, or link to a doc on userIds and anonIds in server side tracking (if we don't have one already, we should)?

**Review by logason — APPROVED** [review]
One suggestion to add more details on userId/anonId setup, probably a separate doc (which can happen in a separate PR).

Lgtm!

File: pages/implementation/reference/typescript.mdx (hunk 3 above)
Comment (alex-tpom6oh) [reply pages/implementation/reference/typescript.mdx:153]: yes, definitely

Comment (alex-tpom6oh) [reply pages/implementation/reference/typescript.mdx:153]: We don't have it now, so I'll do in a separate pr

**Pull request merged**